scripts/common/rook.sh

```diff
@@ -293,25 +295,27 @@ function maybe_cleanup_rook() {
 
         # If upgrade from Rook to OpenEBS without Minio we cannot remove Rook because
         # we do not know if the solution uses or not ObjectStore and if someone data will not be lost
-        if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ -z "$MINIO_VERSION" ]; then
+        if [ -n "$DID_MIGRATE_ROOK_PVCS" ] && [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ -z "$MINIO_VERSION" ]; then
             if [ -z "$DID_MIGRATE_ROOK_OBJECT_STORE" ] || [ "$DID_MIGRATE_ROOK_OBJECT_STORE" != "1" ]; then
                 logWarn "PVC(s) were migrated from Rook but Object Store data was not, as no MinIO version was specified."
                 logWarn "Rook will not be automatically removed without migrating Object Store data."
                 logWarn ""
                 logWarn "If you are sure that Object Store data is not used, you can manually perform this operation"
-                logWarn "by running the remove_rook_ceph task:"                
+                logWarn "by running the remove_rook_ceph task:"
                 logWarn "$ curl <installer>/task.sh | sudo bash -s remove_rook_ceph, i.e.:"
                 logWarn ""
                 logWarn "curl https://kurl.sh/latest/tasks.sh | sudo bash -s remove_rook_ceph"
             fi
         fi
         logFail "Unable to remove Rook."
-        if [ "$DID_MIGRATE_ROOK_PVCS" != "1" ]; then
+        if [ -z "$DID_MIGRATE_ROOK_PVCS" ] || [ "$DID_MIGRATE_ROOK_PVCS" != "1" ]; then
            logFail "Storage class migration did not succeed"
         fi
-        
-        if [ -n "$MINIO_VERSION" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" != "1" ]; then
-           logFail "Object Store migration did not succeed"
+
+        if [ -n "$MINIO_VERSION" ]; then
+           if [ -z "$DID_MIGRATE_ROOK_OBJECT_STORE" ] || [ "$DID_MIGRATE_ROOK_OBJECT_STORE" != "1" ]; then
+              logFail "Object Store migration did not succeed"
+           fi
```

Comment:
this change is not necessary

```suggestion
        if [ -n "$MINIO_VERSION" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" != "1" ]; then
            logFail "Object Store migration did not succeed"
```